diff --git a/node_modules/redux-localstorage-simple/dist/index.js b/node_modules/redux-localstorage-simple/dist/index.js
index 461e14f..6c63c93 100644
--- a/node_modules/redux-localstorage-simple/dist/index.js
+++ b/node_modules/redux-localstorage-simple/dist/index.js
@@ -451,7 +451,11 @@ function load() {
       var key = namespace + namespaceSeparator + state;
       var val = storage.getItem(key);
       if (val) {
-        loadedState = (0, _objectMerge2.default)(loadedState, realiseObject(state, val));
+        if (Object.keys(loadedState).length === 0) {
+          loadedState = realiseObject(state, val);
+        } else {
+          loadedState = (0, _objectMerge2.default)(loadedState, realiseObject(state, val));
+        }
       } else {
         warn_("Invalid load '" + key + "' provided. Check your 'states' in 'load()'. If this is your first time running this app you may see this message. To disable it in future use the 'disableWarnings' flag, see documentation.");
       }
